### PR TITLE
Jointcal test for decam Tickets/dm 6624

### PR DESCRIPTION
**Pull request by parejkoj**
Working version of a jointcal test for decam data, post-VisitInfo.

### CODE REVIEW

File: tests/test_jointcal_decam.py [hunk 1]
```diff
@@ -45,15 +45,17 @@ def setUp(self):
         radius = 3*lsst.afw.geom.degrees
         self._prep_reference_loader(center, radius)
 
-        self.input_dir = os.path.join(data_dir, 'data')
+        self.input_dir = os.path.join(data_dir, 'decam')
         self.all_visits = [176837, 176846]
+        ccdnums = '^'.join(str(x) for x in range(10,19))
```
Comment (mtpatter) [inline tests/test_jointcal_decam.py]: Should there be whitespace after the comma in range?

File: tests/test_jointcal_decam.py [hunk 2]
```diff
@@ -45,15 +45,17 @@ def setUp(self):
...
+        self.other_args = ['ccdnum='+ccdnums,]
```
Comment (mtpatter) [inline tests/test_jointcal_decam.py]: Whitespace after the comma (E231) and also around the + operator (according to the style guide).

File: tests/test_jointcal_decam.py [hunk 3]
```diff
@@ -45,15 +45,17 @@ def setUp(self):
...
 
     @unittest.skipIf(data_dir is None, "validation_data_decam not setup")
```
Comment (mtpatter) [inline tests/test_jointcal_decam.py]: Do you want this to say validation_data_jointcal not setup?

File: python/lsst/jointcal/jointcal.py [hunk 4]
```diff
@@ -159,7 +159,7 @@ def _build_ccdImage(self, dataRef, associations, jointcalControl):
         uRight = afwGeom.Point2I(lLeft.getX() + md.get("NAXIS1")-1, lLeft.getY() + md.get("NAXIS2")-1)
```
Comment (parejkoj) [inline python/lsst/jointcal/jointcal.py]: Fixed.

File: tests/test_jointcal_decam.py (hunk 1 above)
Comment (parejkoj) [inline tests/test_jointcal_decam.py]: Correct. Fixed.

File: tests/test_jointcal_decam.py (hunk 2 above)
Comment (parejkoj) [inline tests/test_jointcal_decam.py]: Fixed.

File: tests/test_jointcal_decam.py (hunk 3 above)
Comment (parejkoj) [inline tests/test_jointcal_decam.py]: Good catch! Yes.

**Pull request merged**